scripts/build_python.sh

```diff
@@ -38,11 +38,64 @@ CHIP_ROOT=$(_normpath "$(dirname "$0")/..")
 OUTPUT_ROOT="$CHIP_ROOT/out/python_lib"
 ENVIRONMENT_ROOT="$CHIP_ROOT/out/python_env"
 
+declare chip_detail_logging=false
+declare chip_mdns="minimal"
+
+help() {
+
+    echo "Usage: $file_name [ options ... ] [ -chip_detail_logging ChipDetailLoggingValue  ] [ -chip_mdns ChipMDNSValue  ]"
+
+    echo "General Options:
+  -h, --help                Display this information.
+Input Options:
+  -d, --chip_detail_logging ChipDetailLoggingValue          Specify ChipDetailLoggingValue as true or false.
+                                                            By default it is false.
+  -m, --chip_mdns           ChipMDNSValue                   Specify ChipMDNSValue as platform or minimal.
+                                                            By default it is minimal.
+"
+}
+
+file_name=${0##*/}
+
+while (($#)); do
+    case $1 in
+        --help | -h)
+            help
+            ;;
+        --chip_detail_logging | -d)
+            chip_detail_logging=$2
+            ;;
+        --chip_mdns | -m)
+            chip_mdns=$2
+            ;;
+        -*)
+            help
+            echo "Unknown Option \"$1\""
+            ;;
+    esac
+    case $3 in
```

Comment:
Why is the second `case` needed? `shift` in the loop causes that subsequent arguments become `$1` in subsequent iterations, so they should be parsed with the first `case.

Reply:
No the second case is not needed now, removed👍